js/mqtt/mqttHandler.js

```diff
@@ -58,6 +61,23 @@ class MQTTHandler extends Handler {
         break;
     }
   }
+
+  _queueSubscription = (topic, callback) => {
```

Comment:
Unless I'm missing something, I think having a queue in both files is a little redundant? I would only queue things in the websocket.js file.